ui/src/main.ts

```diff
@@ -33,6 +44,7 @@ export {
   NewDateInput,
   DateRangeInput,
   Pagination,
+  StepFormComponent,
```

Comment:
Maybe this could be called `StepForm` only?

Reply:
There is already a StepForm component (it's the base form component used by each steps :wink: )